src/utils.rs

```diff
@@ -1 +1,21 @@
+use std::fs::File;
+use std::io::Read;
+use std::path::Path;
+use serde::de::DeserializeOwned;
+
+
 pub type BoxError = anyhow::Error;
+
+pub trait FromTomlFile: DeserializeOwned {
```

Comment:
Why does it have to be a trait instead of helper method?

Reply:
Because it is a plain copy from https://github.com/Sovereign-Labs/sovereign/pull/293

And I copied, because I haven't decided where this kind of trait suppose to live. Maybe there's already a package for that